agent/config/builder.go

```diff
@@ -1198,7 +1198,7 @@ func (b *Builder) expandAddrs(name string, s *string) []net.Addr {
 
 	x, err := template.Parse(*s)
 	if err != nil {
-		b.err = multierror.Append(b.err, fmt.Errorf("%s: error parsing %q: %s", name, s, err))
+		b.err = multierror.Append(b.err, fmt.Errorf("%s: error parsing %s: %s", name, *s, err))
```

Comment:
Do the errors go away just with changing `s` to `*s`? If so I think that is probably the way to go. I believe %q was being used to have go wrap that string with quotes for us.